pkg/controller/service/servicecontroller.go

```diff
@@ -741,6 +741,7 @@ func (s *ServiceController) lockedUpdateLoadBalancerHosts(service *api.Service,
 		return nil
 	}
 
+	hosts := hostsFromNodeList(nodes)
```

Comment:
Function call signature does not match: hostsFromNodeList(nodes)
Function call definition: func HostsFromNodeList(list *api.NodeList) []string {}

Definition should probably be camel case, hostsFromNodeList